### PR TITLE
feat(nginx): Allow sending nginx X-Request-Id and X-Correlation-Id headers.

### PR DESCRIPTION
**Pull request by robholland**
This is useful for correlating traffic when using microservices. If this is accepted I'll add a PR to adjust the logging format so that this data is available in the logs.

### CODE REVIEW

**Comment by deis-admin** [comment]
Thanks for the contribution! Please ensure your commits follow our [style guide](http://docs.deis.io/en/latest/contributing/standards/#commit-style-guide). This code will be tested once a Deis maintainer reviews it.

**Comment by mboersma** [comment]
Jenkins, add to whitelist.

File: model/model.go [hunk 1]
```diff
@@ -49,6 +49,7 @@ type RouterConfig struct {
 	EnforceWhitelists        bool        `key:"enforceWhitelists" constraint:"(?i)^(true|false)$"`
 	DefaultWhitelist         []string    `key:"defaultWhitelist" constraint:"^((([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\\/([0-9]|[1-2][0-9]|3[0-2]))?(\\s*,\\s*)?)+$"`
 	WhitelistMode            string      `key:"whitelistMode" constraint:"^(extend|override)$"`
+	RequestIDs               bool        `key:"RequestIDs" constraint:"(?i)^(true|false)$"`
```
Comment (kmala) [inline model/model.go]: can you add it to the readme under the annotations section https://github.com/deis/router/blob/master/README.md

Comment (robholland) [inline model/model.go]: Updated in b16b41c0e033fce4d751e9207c79f1c346947b45

**Comment by mboersma** [comment]
This LGTM, but @krancour should have the last word before it gets merged.

**Pull request merged**